types/budget.d.ts

```diff
@@ -38,12 +38,19 @@ declare global {
         metric: TimingMetric;
         /** Budget for timing measurement, in milliseconds. */
         budget: number;
-        /** Tolerance, i.e. buffer, to apply to a timing budget. Units: milliseconds. */
-        tolerance?: number;
       }
 
+      /**
+       * This indicates how measurements taken over multiple runs should be interpretted; the default is 1.
+       * 'Median' mode fails if the median run fails.
+       * 'Optimistic' mode fails only if all runs fail. Put another way, this passes if any run passes.
+       * 'Pessimistic' mode fails if any of the runs fail. Put another way, this only passes if all runs pass.
+       *  This can be supplied if only 1 run is used, but it is superfluous because all strategies would yield the same result.
+       */
+      export type AggregationMethod = 'median' | 'optimistic' | 'pessimistic';
```

Comment:
seems like we don't actually leverage this at the moment, is it missing from `Budget` interface or did we land on postponing this?

Reply:
Oops, I thought I had removed this when I was removing the `runs` related stuff. Yep, it's not needed.